Add obs-clippings extension

trevware opened the pull request:
## Description

See README.

## Screencast

[Vimeo](https://vimeo.com/835944114) -- Note that the content was too long for AI to summarize in this example, but there is a corresponding toast to cover these scenarios.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- There is no validation, I can click `Create Clipping` without anything in the form and get a success toast.

- Do I really need a subfolder?

- I tried to make a clipping with and without ticking `Extract Content and Summarize with AI` 

- it seems like both ended up with AI 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

trevware commented:
@pernielsentikaer Ah, didn't catch some things.

- Added form validation if form is not yet complete.
- Made sub-folder optional.
- Added additional messaging if and when user is not extracting content.
- Added additional validation for titles and tags such that it is correctly parsed by the OS.

This should fix your AI toggle issue. I can no longer replicate it. If disabled, the messaging in the note created states that there is no AI Summary nor is there any extracted content. If enabled, runs as intended!

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
